src/Stormpath/Resource/Account.php

```diff
@@ -207,4 +208,22 @@ public function delete() {
 
         $this->getDataStore()->delete($this);
     }
+
+    public function createApiKey($options = array())
+    {
+        $apiKeyOptions = new ApiKeyEncryptionOptions($options);
+        $options = array_merge($options, $apiKeyOptions->toArray());
+
+        $apiKey = $this->getDataStore()->instantiate(Stormpath::API_KEY);
+
+        $apiKey = $this->getDataStore()->create($this->getHref() . '/' . ApiKey::PATH,
+            $apiKey, Stormpath::API_KEY, $options);
+
+        if ($apiKey)
+        {
+            $apiKey->setApiKeyMetadata($apiKeyOptions);
+        }
+
+        return $apiKey;
+    }
```

Comment:
Alright, but we still need to add the api key collection to the account resource.